doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4631,6 +4631,31 @@ The issues in this section are related to :ref:`nrfxlib:crypto`.
 
 .. rst-class:: v3-1-1 v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-0-nRF54H20-1 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0
 
+NCSDK-35975: The CRACEN driver outputs wrong results for PBKDF2 using HMAC
```

Comment:
Please add a comment  on PR https://github.com/nrfconnect/sdk-nrf/pull/25844 for this change and remove this entry from this PR. Known issues are tracked on PR 25844. FYI - @peknis .